### ISSUE TITLE
The example winFun in your suggestion.

### ISSUE THREAD

**Issue by KyuhoCNX** [AndyPL22/ForestTools#14]
I'm using this package for my master's research,
I developed window size functions with our own data, 
but somehow, your example function worked better for finding tree-top ponts.  

so I want to know the source of the equation, where you could bring the equation.

by the way, your equation ( function(x){x * 0.05 + 0.6} ) in 'Canopy analysis in R using ForestTools'-2020.05.06 
and the the equation (=function(x){x * 0.06 + 0.5}) in Package description,
are different. Which one is what you intended?

**Comment by andrew-plowright** [comment]
Hi @KyuhoCNX , sorry to disappoint but those are arbitrary functions that I used when working on some sample data. They're not based on any research.

**Comment by NatKatUP** [comment]
Hey @KyuhoCNX I also used these methods in my master's research and discussed the use of different search functions for different canopy structure types in my resulting paper. This might be of interest to you: https://doi.org/10.3390/rs11121447

**Comment by andrew-plowright** [comment]
@NatKatUP nice work. Can I link to that article on the package's main page? I've been meaning to compile articles that use the package and that can help people calibrate the algorithm.

**Comment by NatKatUP** [comment]
@andrew-plowright Please, go ahead! I would actually really appreciate that.

**Comment by KyuhoCNX** [comment]
@andrew-plowright @NatKatUP Oh, Thank you for your rapid responses!
It really helps me.

but have another question to @NatKatUP.
I briefly checked your paper with four equations,
and you described that you experimented with various set-ups and found that generally, a much narrower function works better,

did you try different numbers in the simple linear equation and then selected the best thing?
or were there any references that helped you choosing potential slopes and intercepts?

**Comment by NatKatUP** [comment]
@KyuhoCNX I first tried using empirical relationships between tree size and crown diameter measured in GIS but they did no work as well. There is still a strong relationship between the narrowness of crowns and the ideal search function it's just narrower than you would think. As discussed in the paper this is not a problem for most single stemmed conifers as I found them in my study area but leads to problems when crowns are very heterogeneously structured within your dataset (like occasional wide crowns /multi stem trees / leaning trees).

But yes, try and error  helps finding the best function for your canopy structure :)
